webpack.config.js

```diff
@@ -32,6 +32,9 @@ var webpack = require('webpack'),
         selector: '[ng-app]'
       })
     }),
+    new webpack.ProvidePlugin({
```

Comment:
I'm not really wild about the webpack ProvidePlugin, since it magically provides that module everywhere. Kind of similiar to global variables but for modules. Explicitly importing Rx seems like a better way. Most components won't need to have the whole Rx module any but a very specific subset of operators.
Tooling that recognizes imports probably won't understand implicit webpack modules and will show errors all over the place (possibly even for linting tools).